coulomb-tests/shared/src/test/scala/coulomb/validators.scala

```diff
@@ -29,18 +28,18 @@ object CoulombValidators {
   }
 
   implicit class WithQuantityShouldMethods[V, U](q: Quantity[V, U])(implicit
-      ttV: WeakTypeTag[V],
-      ttU: WeakTypeTag[U],
+      ttV: UnitTypeName[V],
```

Comment:
we use `UnitTypeName` as a local type class to abstract reading the name of the unit